src/ray/stats/metric_exporter.h

```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RAY_METRIC_EXPORTER_H
+#define RAY_METRIC_EXPORTER_H
```

Comment:
Can you replace it with `pragma once`? That's the new convention.

Reply:
We don't need 
```
#ifndef RAY_METRIC_EXPORTER_H
#define RAY_METRIC_EXPORTER_H
```
anymore. Ex) https://github.com/ray-project/ray/blob/0ff24ec8dcc4b744981bfd544e70bc748d34c4e4/src/ray/raylet/node_manager.h#L15